hms-app/src/components/input-forms/HackathonForm.tsx

```diff
@@ -207,9 +210,9 @@ function HackathonForm(props: IProps) {
             Description
           </Title>
           <RichTextEditor
-            value={DescriptionValue}
+            value={descriptionValue}
             onChange={onChange}
-            style={{ minHeight: 225 }}
+            id='rte'
```

Comment:
the ID could be a good bit more descriptive (and unique), `hackathonDescriptionEditor` or so